### ISSUE TITLE
Keep in mind that we may need to change the "SACM" prefix

### ISSUE THREAD

**Issue by adammontville** [sacmwg/draft-ietf-sacm-terminology#85]
Generally, it seems, terms we define should not contain the "SACM" prefix, which is the name of the working group. This is not an immediate need, but we may need to address this in the future. This issue exists so that we are less likely to forget.

**Comment by henkbirkholz** [comment]
Yes. I would urge to put this topic on the agenda for the next meeting, please.
